### PR TITLE
Name, tag name, address and email string may be too long to display

### LINKED ISSUE #97: Name, tag name, address and email string may be too long to display
Name, Address, email, Tag names may be too long to display. Should set a reasonable upper limit to the size of the strings.

Image

In (Singapore Corppass)[https://public.cloud.myinfo.gov.sg/edh/edh-tuo-specs.html], the name of a user is limited to 100 characters.

Email in Official RFC Standards is up to 254 characters

Tag Name we want to limit it to 40 Characters, enough for identifying most premium packages and tags

Address just follow convention and keep to 254 characters.

### PR DESCRIPTION
**Pull request by Huffle-Buffle**
Fixes #97

### CODE REVIEW

File: src/test/java/seedu/address/model/person/AddressTest.java [hunk 1]
```diff
@@ -42,48 +46,84 @@ public void isValidAddress() {
...
         Address a2 = new Address("123 Main St #05-05");
-        // equals implies same hashCode
+        // equals implies same hashCodes
         assertTrue(a1.equals(a2));
         org.junit.jupiter.api.Assertions.assertEquals(a1.hashCode(), a2.hashCode());
     }
 
+
+    @Test
+    public void exceedsMaxLength_returnsFalse() {
```
Comment (robinlogic) [inline src/test/java/seedu/address/model/person/AddressTest.java]: Great test additions! I was hoping to see an Equivalence Boundary Case, and here it is.
